### ISSUE TITLE
Anime playing wrong videos

### ISSUE THREAD

**Issue by ShanSharma08** [chirag-droid/animeflix#145]
Certain anime are playing the wrong videos
For example, Code geass:Hangyaku no Lelouch (TV) is playing Code Geass: Hangyaku no Lelouch I - Koudou (MOVIE), same with Code geass:Hangyaku no Lelouch R2 (TV) playing the omake flash version (SPECIAL)
Most probably happening because both have very similar names so its fetching the other one's data.
The episode list works fine but only the player plays the wrong video
For now, i only noticed this for code geass, there might be other anime series facing the same issue

Go to Code geass:Hangyaku no Lelouch / Code geass:Hangyaku no Lelouch R2
Click on watch now or an episode
See that its not playing the correct one

Expected: The player plays the tv series , not the movie

Desktop
 - OS: Windows 10
 - Chrome
 - Version Dev 103

**Comment by TomRomeo** [comment]
Yes, this probably happens because the wrong stream is selected for some Anime with similar names. iirc the same happens with Initial D.

Maybe we can implement a 'change stream' button to select the right stream manually if the automatic detection fails.

**Comment by riimuru** [comment]
I will push a fix for this issue later. It won't fix all animes but it should fix most of them. After that @TomRomeo and @chirag-droid should implement the "wrong title? " feature.

**Comment by chirag-droid** [comment]
Now I have implemented the state management using redux. So now it should be easy to implement this feature
